lessons/intro/pyqt/index.md

```diff
@@ -90,7 +90,7 @@ O Qt, PyQt a PySide
 
 [PyQt](https://riverbankcomputing.com/software/pyqt) je knihovna, která umožňuje použít Qt z Pythonu.
 Na rozdíl od samotného Qt je licencovaná pod [GNU GPL v3](https://www.gnu.org/licenses/gpl-3.0.en.html), která (stručně řečeno) vyžaduje, aby programy napsané s použitím PyQt byly šířeny pod stejnou licencí a se zdrojovým kódem.
-Tedy: kdokoliv, kdo dostane kopii programu, musí mít možnost dostat odpovídající zdrojový kód, a má možnost tento kód dál šířit pod stejnou licencí.
+Tedy: kdokoliv někdo dostane kopii programu, musí mít možnost dostat odpovídající zdrojový kód a má možnost tento kód dál šířit pod stejnou licencí.
```

Comment:
Předtím to bylo dobře. případně se to dá použít jak to je teď, ale je třeba změnit „kdokoliv“ na „kdykoliv“, jinak to nedává smysl.